ML_experiments.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from skimage.util.shape import view_as_windows
+
+def create_train_test_validation(dictionary, pos_key, neg_key, percent_train, percent_test, percent_validation):
+    """
+        Description: creates a training, testing, and validation dictionary for binary data  
+
+        Args:
+            dictionary -- dict (Dictionary of data sorted into pos_key and neg_key: 
+                                dictionary = {pos_key: [data, data,...], neg_key: [data, data, ...]})
+            pos_key -- String (Key for the positive data samples in dictionary)
+            neg_key -- String (Key for the negative data samples in dictionary)
+            percent_train -- double ([0-1] Percentage of data to go to training set)
+            percent_test -- double (0-1] Percentage of data to go to testing set)
+            percent_validation -- double ([0-1] Percentage of data to go to validation set)
+
+        Return:
+            train -- dict (Dictionary with percent_train of original dictionary)
+            test -- dict (Dictionary with percent_test of original dictionary)
+            validation -- dict (Dictionary with percent_validation of original dictionary)
+        """
+    train = {pos_key: [], neg_key: []}
+    test = {pos_key: [], neg_key: []}
+    validation = {pos_key: [], neg_key: []}
```

Comment:
Is there a benefit to create these here and could you just create them in the splitting_up_data function?